app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java

```diff
@@ -137,4 +141,9 @@ protected void onDestroy() {
         mNavigationView = null;
         super.onDestroy();
     }
+
+    public void forceCrash(View view) {
```

Comment:
I doubt you'll ever use this again. Since the crash test was successful (as shown on the Crashlytics dashboard), please remove this :)